internal/system/agent/executor/compose.go

```diff
@@ -119,37 +118,27 @@ func RunDockerComposeCommand(service string, dockerComposeService string) {
 
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		log.Fatal(fmt.Sprintf("Call to docker-compose up -d failed with %s\n", err))
+		log.Printf(fmt.Sprintf("Call to docker-compose up -d failed with %s\n", err.Error()))
+		// agent.LoggingClient.Error("Call to docker-compose up -d failed with %s\n", err.Error())
 	}
-
-	// composeOutput := string(out)
-	// fmt.Sprintf("For the micro-service {%v}, we got this docker-compose output:\n {%v}", service, composeOutput)
 	println(out)
 
 	if ! WasDockerContainerComposeStarted(service) {
-		log.Fatal(fmt.Sprintf("The container for {%v} was NOT started!" + service))
+		log.Printf(fmt.Sprintf("The container for {%s} was NOT started!" + service))
```

Comment:
This line should be

`log.Printf(fmt.Sprintf("The container for {%s} was NOT started!" , service))`

Reply:
Shouldn't it just be 
```go
log.Printf("The container for {%s} was NOT started!\n", service)
```
(also noting the trailing `\n`)